org.eclipse.ice.build/pom.xml

```diff
@@ -1,31 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
 
-  <parent>
-  	<groupId>org.eclipse.ice</groupId>
-  	<artifactId>org.eclipse.ice.parent</artifactId>
-  	<version>2.2.1-SNAPSHOT</version>
-  	<relativePath>../org.eclipse.ice.parent/pom.xml</relativePath>
-  </parent>
-  <groupId>org.eclipse.ice</groupId>
-  <artifactId>org.eclipse.ice.build</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <packaging>pom</packaging>
+	<parent>
```

Comment:
This versioning looks wrong. Everything should be 3.0. Also, I'm not sure why we are using the old parent.